arm-asazure/2016-05-06/swagger/asazure.json

```diff
@@ -0,0 +1,551 @@
+      "get": {
+        "description": "Gets result of the specified operation. The API return the HTTP status code and a response body",
+        "tags": [ "RpProxy" ],
+        "operationId": "RpProxy_GetOperationStatus",
+        "parameters": [
+          {
+            "name": "operationId",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "x-ms-skip-url-encoding": true,
+            "description": "Identifier for the operation for which to get status"
+          },
+          {
+            "name": "location",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "x-ms-skip-url-encoding": true,
+            "description": "Region in which the server operation was executed"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": { "$ref": "#/definitions/OperationDetail" }
+          },
+          "202": {
+            "description": "Accepted",
+            "schema": { "$ref": "#/definitions/OperationDetail" }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.AnalysisServices/locations/{location}/operationresults/{operationId}": {
+      "get": {
+        "description": "Gets result of the specified operation. The API return only the HTTP status code and does not have a response body",
+        "tags": [ "RpProxy" ],
+        "operationId": "RpProxy_GetOperationResult",
+        "parameters": [
+          {
+            "name": "operationId",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "x-ms-skip-url-encoding": true,
+            "description": "Identifier for the operation for which to get status"
+          },
+          {
+            "name": "location",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "x-ms-skip-url-encoding": true,
+            "description": "Region in which the server operation was executed"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK"
+          },
+          "202": {
+            "description": "Accepted"
+          },
+          "204": {
+            "description": "No Content"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "RpServerUpdateRequest": {
+      "description": "Update request specification",
+      "type": "object",
+      "properties": {
+        "location": {
```

Comment:
- Since this is optional can you tell us what will happen if the user
    - changes the value of location from "West US" -> "East US"? What will the server do (accept the new value, ignore it or throw an error) ?
  - does not provide the location in the update scenario?